docs/Controller Guide.md

```diff
@@ -0,0 +1,100 @@
+# NMOS Controller Implementation Guide
+
+## Scope
+This document is intended as a guide for implementers or users of Controllers within NMOS-enabled networked media systems. The document defines what a Controller is and outlines the requirements of a Controller with respect to each existing NMOS specification through references to the relevant sections of those documents.
+
+The document focuses primarily on guidance for the core NMOS specifications of IS-04, IS-05 and BCP-003-01. However, this is a living document and it is intended that fuller guidance for other NMOS specifications be added in future.
```

Comment:
I don't think there is an official list of `core` specifications. Should we just say:
The document focuses primarily on guidance for the following NMOS specifications:
* IS-04
* IS-05
* BCP-003-01

Reply:
Agreed. I have made that change.